backend/src/database/repositories/segmentRepository.ts

```diff
@@ -313,6 +347,10 @@ class SegmentRepository extends RepositoryBase<
       },
     )
 
+    records.forEach((row) => {
+      row.activityChannels = row.activityChannels[0]
```

Comment:
Wait a sec, this doesn't look right. This way you only get a single platform, because 
```json_agg(sac."activityChannels")``` will give you a json similar to 
```
[
  {"platform1" : ["channel1", "channel2", "channel3"]},
  {"platform2" : ["channel1", "channel2", "channel3"]}
]
```